templates/index.html

```diff
@@ -1,7 +1,7 @@
 <div class="invite-colleagues__outer" data-trackable="invite-colleagues">
 	<div data-o-grid-colspan="10 center">
 		<div class="o-grid-row">
-			<div data-o-grid-colspan="10 M8 XL6 center">
+			<div data-o-grid-colspan="10 M9 XL5 center">
```

Comment:
Is this to fix the way it looks on the demo, or within the app? I tried it in `next-corp-signup` and it breaks the design 😕

Reply:
within the app - to stop 'Financial Times' splitting onto two lines. But obvs where it splits depends on the resolution so I tried to (somewhat arbitrarily!) optimise for our desktop screens here. Will take another look...

Reply:
If you want to do that, you can wrap it in a span with a class that uses `white-space: nowrap;`